python/ray/air/integrations/wandb.py

```diff
@@ -671,9 +672,12 @@ def log_trial_result(self, iteration: int, trial: "Trial", result: Dict):
 
     def log_trial_save(self, trial: "Trial"):
         if self.upload_checkpoints and trial.checkpoint:
-            self._trial_queues[trial].put(
-                (_QueueItem.CHECKPOINT, trial.checkpoint.dir_or_data)
-            )
+            if _use_storage_context():
+                checkpoint_root = trial.checkpoint.path
```

Comment:
Do we need to do the same local filesystem check here?

Reply:
I think it makes sense, added. Thanks!